### PR TITLE
Add finalized agenda and abstracts

### PR DESCRIPTION
**Pull request by mrmundt**
This PR adds the full finalized agenda with abstracts (**NOTE**: speakers are in alphabetical order by presenter's last name).

This also adds some corrections on dates, adds the workshop date, and strikes-out past deadlines.

### CODE REVIEW

File: pages/abstracts.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+view on scientific library development and surfaces opportunities for future
+studies into effective strategies for RSEs.
+
+---
+
+### How to Grow Diverse and Sustainable Teams through Mentorship
+
+*Caleb Jackson*
+
+Abstract	The chief question I will address is: in a sea of open-source
```
Comment (jbteves) [inline pages/abstracts.md:78]:
```suggestion
The chief question I will address is: in a sea of open-source
```

**Review by jbteves — APPROVED** [review]
Looks good to me.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,6 @@
 
 This is the website landing page for the RSE-eScience-2022 workshop to be
 held in conjunction with [eScience 2022](https://www.escience-conference.org/2022/)
-in Salt Lake City, Utah, October 11-14, 2022.
+in Salt Lake City, Utah, October 10-14, 2022.
```
Comment (danielskatz) [inline README.md:7]: Maybe we should change the date to just the data of the workshop, not the date of the whole conference

File: index.html [hunk 3]
```diff
@@ -10,7 +10,7 @@
 <div class="main-explain-area jumbotron">
   <h2><center>To be held as part of
   <a href='https://www.escience-conference.org/2022/'>eScience 2022</a>,
-  Salt Lake City, Utah, October 11-14, 2022.
+  Salt Lake City, Utah, October 10-14, 2022.
```
Comment (danielskatz) [inline index.html:13]: Maybe we should change the date to just the data of the workshop, not the date of the whole conference

**Comment by danielskatz** [comment]
Is there a preview available?

**Comment by mrmundt** [comment]
> Is there a preview available?

No, this repo doesn't have that enabled. But I based it off the 2021 virtual workshop website: https://us-rse.org/virtual-workshop-2021/agenda/

**Pull request merged**